helios-docs/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'ank-gradle-plugin'
+
+dependencies {
+    compile project(":helios-core")
+    compile project(":helios-optics")
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+
+    kapt "io.arrow-kt:arrow-annotations-processor:$arrowVersion"
+    kapt project(':helios-meta')
+    kapt project(':helios-meta-compiler')
+    kapt project(':helios-dsl-syntax-compiler')
+    implementation "io.arrow-kt:arrow-ank:$arrowVersion"
+}
+
+ank {
```

Comment:
People don't need to use ank to use Helios. Why is this here?

Reply:
Is the build to the docs module, I created to use Ank in the docs

Reply:
my bad, I thought this change was in the README file.